### PR TITLE
Removing Docker references 

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@vikram-redhat, @smarterclayton, PTAL

File: _topic_map.yml [hunk 1]
```diff
@@ -283,7 +283,7 @@ Topics:
 - Name: Disconnected installation
   File: disconnected_install
   Distros: openshift-enterprise
-- Name: InInstalling a stand-alone deployment of OpenShift container registry
+- Name: InInstalling a stand-alone deployment of OpenShift container image registry
```
Comment (vikram-redhat) [inline _topic_map.yml]: Looks like a long standing typo :)

**Comment by vikram-redhat** [comment]
@kalexand-rh I am happy with it. If you don't hear from anyone else, go ahead and merge for 3.11 release and close the originating PR.

File: _topic_map.yml (hunk 1 above)
Comment (kalexand-rh) [reply _topic_map.yml]: Luckily it's just in 3.11, so this PR will cover it. :)

**Pull request merged**